libflux/go/libflux/analyze.go

```diff
@@ -68,3 +68,10 @@ func Analyze(astPkg *ASTPkg) (*SemanticPkg, error) {
 	runtime.SetFinalizer(p, free)
 	return p, nil
 }
+// TODO(faith): fix comment; create the buffer, pass it in and goes takes care of freeing data
+func EnvStdlib() []byte {
+	var buf C.struct_flux_buffer_t
+	C.flux_get_env_stdlib(&buf)
```

Comment:
You'll want to call `flux_free` to release the buffer allocated by Rust, similar to how `MarshalFB` does above.

Reply:
Yeah thanks! I was waiting on this update so I'll go ahead and conform to how we're doing this now.

Reply:
I think it should be safe... If you are calling `GoBytes` here, then it will do a copy, and it will be safe to free the memory that you got from Rust.

Reply:
Yeah, changed this. Doesn't seem to be affecting the way the lookup performs.